core/src/main/thrift/tabletserver.thrift

```diff
@@ -348,6 +350,16 @@ service TabletClientService extends client.ClientService {
     7:i64 requestTime
   )
 
+  // since 2.1
+  oneway void unload(
+      5:trace.TInfo tinfo
+      1:security.TCredentials credentials
+      4:string lock
+      2:data.TKeyExtent extent
+      6:string goal
+      7:i64 requestTime
```

Comment:
These fields could be numbered sanely, if you're creating a new function. The old fields were numbered that way because it evolved over time.